src/query/execute_statement.c

```diff
@@ -18421,180 +18448,11 @@ get_dblink_password_decrypt (PARSER_CONTEXT * parser, const char *passwd_cipher,
-	{
-	  start = (chk_time->tv_usec >> 3) % start;
-	}
-    }
-  *p = (unsigned char) start;
-  p++;
-
-  for (i = 0; i < start; i++)
-    {
-      p[i] = tmpx[rand () % tmpx_len];
-    }
-  p += start;
-
-  if (pwdlen > 0)
-    {
-      shift %= pwdlen;
-      if (shift == 0)
-	{
-	  shift++;
-	}
 
-      // conceptual example) password --> ordpassw
-      for (i = 0; i < pwdlen; i++)
-	{
-	  p[((i + shift) < pwdlen) ? (i + shift) : ((i + shift) - pwdlen)] = passwd[i];
-	}
-
-      // conceptual example) ordpassw --> pseqbttx
-      shift = ((chk_time->tv_usec & 0x7A5A) % 127) + 1;
-      for (i = 0; i < pwdlen; i++)
-	{
-	  p[i] = ((p[i] + shift) < 128) ? (p[i] + shift) : (p[i] + shift - 128);
-	}
-
-      p += pwdlen;
-    }
-
-  pwdlen = p - ((unsigned char *) confused);
-  checksum = 0x7C;
-  p = (unsigned char *) confused;
-  for (i = 0; i < pwdlen; i++)
-    {
-      checksum += p[i];
-    }
-  p[pwdlen++] = checksum;
-
-  /* Adjust the length so that it is a multiple of 3.
-   * This is to prevent the '=' character from appearing in the base64 conversion result.  */
-  while (pwdlen % 3)
-    {
-      p[pwdlen++] = checksum;
-    }
-
-  p[pwdlen] = '\0';
-  return pwdlen;
-}
-
-/*
- * reverse_shake_dblink_password ()  : Extract the raw password from the re-created password.
- *
- * return	       : NO_ERROR or error code.
- * confused(in)	       : Password created via shake_dblink_password(). This is binary.
- * length(in)          : lelgth of "confused" 
- * passwd(out)         : Raw password.
- * 
- * Remark: 
- *      
- */
-static int
-reverse_shake_dblink_password (char *confused, int length, char *passwd)
-{
-  int i, pwdlen;
-  int shift;
-  unsigned char *p;
-  unsigned char checksum;
-
-  p = (unsigned char *) confused;
-  memcpy (&shift, p, sizeof (int));
-  p += sizeof (int);
-  pwdlen = *p;
-  p++;
-  p += (*p + 1);
-
-  if (length < pwdlen)
-    {
-      return ER_FAILED;
-    }
-
-  if (pwdlen > 0)
-    {
-      shift %= pwdlen;
-      if (shift == 0)
-	{
-	  shift++;
-	}
-      // conceptual example) pseqbttx --> ordpassw 
-      for (i = 0; i < pwdlen; i++)
-	{
-	  passwd[i] = p[((i + shift) < pwdlen) ? (i + shift) : ((i + shift) - pwdlen)];
-	}
-
-      memcpy (&shift, confused, sizeof (int));
-      shift = ((shift & 0x7A5A) % 127) + 1;
-      // conceptual example) ordpassw --> password
-      for (i = 0; i < pwdlen; i++)
-	{
-	  passwd[i] = ((passwd[i] - shift) >= 0) ? (passwd[i] - shift) : (passwd[i] - shift + 128);
-	}
-    }
-  passwd[pwdlen] = '\0';
-  p += pwdlen;
-
-  pwdlen = p - ((unsigned char *) confused);
-  checksum = 0x7C;
-  p = (unsigned char *) confused;
-  for (i = 0; i < pwdlen; i++)
-    {
-      checksum += p[i];
-    }
-
-  do
-    {
-      if (p[pwdlen] != checksum)
-	{
-	  return ER_FAILED;
-	}
-    }
-  while (++pwdlen % 3);
-
-  return NO_ERROR;
-}
 
 #ifndef NDEBUG
 void
-ciper_func_test (PARSER_CONTEXT * parser)
+ciper_func_test (PARSER_CONTEXT * parser, bool is_timecheck)
```

Comment:
This ciper_func_test() function don't need PR check.
The code will be deleted after the DBLINK function is complete.